### PR TITLE
chore: CI silent jest reporting

### PR DESCRIPTION
**Pull request by montezume**
Let's make jest silent like we have it on mc.

I think this is all we need? Let me know if it's missing something.

Closes [49](https://github.com/commercetools/ui-kit/issues/49)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ cache:
   directories:
     - node_modules
 
-script: yarn run jest --projects jest.*.config.js --maxWorkers=4
+script: yarn run jest --projects jest.*.config.js --maxWorkers=4 --reporters jest-silent-reporter
```
Comment (tdeekens) [inline .travis.yml]: Don't we have to add it as a dep? Otherwise it will, if at all, be only there through somebody else requiring it.

Comment (montezume) [reply .travis.yml]: @tdeekens it's a dep of `commercetools-frontend/jest-preset-mc-app` though

Comment (tdeekens) [reply .travis.yml]: I don't have too strong opinions then. I don't expect us to drop our preset but I often feel direct deps can't hurt.

Comment (dferber90) [reply .travis.yml]: I'd vote for a direct dep as we use it directly.

**Pull request merged**